### ISSUE TITLE
Keyboard navigation doesn't work on mobile

### ISSUE THREAD

**Issue by Ariane-B** [biati-digital/glightbox#91]
**Describe the bug**
None of the keyboard events trigger on anything that registers as a mobile device. No errors or anything.

**Are you able to reproduce the bug in the demo site**
Yes

**To Reproduce**
Load site in your browser's inspector's mobile simulation mode (tested on Firefox and Chrome).
Open a popup.
Try to navigate with your keyboard

**Expected behavior**
It would've been nice if keyboard navigation worked even on mobile. To be honest, right now this only affects development when I load a page in mobile simulator mode and then continue with my business in full mode and keyboard navigation still doesn't work.

But I can imagine several fringe cases where someone who uses a tablet/phone in a desktop-like scenario (e.g. Samsung DeX) would have this issue.

**Screenshots**
Not really applicable, no.

**Desktop:**
 - OS: macOS
 - Browser: Firefox, Chrome
 - Version FF 73.0.1, C 79.0.3945

**Smartphone:**
I actually tested it with an OTG keyboard on my phone.
 - Device: OnePlus 6
 - OS: OxygenOS 10.3.1 (Android 10)
 - Browser: Firefox
 - Version: 68.5.0

**Comment by biati-digital** [comment]
To be honest I don't think this is necessary, although activating the keyboard on mobile only takes deleting 1 line of code, I don't see the point.

**Comment by Ariane-B** [comment]
In my opinion, when looking at this sort of thing, it's always better to ask yourself: "do I *need* to disable it?" rather than "what's the point of it?"

I just told you one case where this behavior is hostile to the user -- me, the developer -- as well as fringe cases where non-developers might have an issue with this. Whichever way you're doing this, it's wrong for different reasons.

If you detect user agent to determine whether it's a mobile device, there are plenty of resources that will explain why user agent sniffing is usually a bad idea. For example, [this one](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent).

If you're detecting whether the device has a touchscreen, please keep in mind that several PCs have touchscreens, and nearly all tablets can be converted into a workstation with mouse and keyboard.

Disabling things just because you "don't see the point" of them is a great way to accidentally cause accessibility issues. Things should just work no matter what combination of peripherals we use them with.

**Comment by biati-digital** [comment]
I fail to see "accessibility issues" if the keyboard navigation is not enabled on mobile devices creating an "hostile behavior" for you the developer.

When I asked for a use case I meant something for the rest 99.9% of non developers that will interact with the Lightbox. I know you want it and I know why you want it but that's not enough because even you said it 

> **To be honest, right now this only affects development**

Please note I'm not trying to be rude or anything is just that If you see some of the closed issues you'll see that people ask for things that may apply to their project or current necessities but it will not be helpful for everyone else or the vast majority, I always have to ask a valid reason to do a change in the library. In the past when I've asked for a reason I've received responses like this:

> **it's quite a difficult answer why we would need this...**

So yeah, as you put it, I do need to ask what's the point of it? because I will not invest my free time doing things that will not be helpful for the vast majority or it's not even clear why we would need this.

If you disagree with this that's fine, you are welcome to fork this project and make all the changes you want or need.

I know about the user agent, that part of the code is really old, I have limited time to work on this library and there has not been a single issue posted about this so I've posponed it for quite some time. if you have time and interest pull requests are welcome.
